Tests/test_NCBIXML.py

```diff
@@ -1866,6 +1854,1681 @@ def test_xml_2218L_rpsblast_001(self):
+                              <Parameters_filter>L;</Parameters_filter>
+                            </Parameters>
+                          </BlastOutput_param>
+                          <BlastOutput_iterations>
+                          </BlastOutput_iterations>
+                        </BlastOutput>
+                        """)
+
+        self.assertRaises(ValueError, NCBIXML.read, handle)
+        handle.close()
+
+    def test_xml_2226_tblastn_007_err(self):
+        """Parse TBLASTN 2.2.26+."""
+        handle = StringIO("""this is an empty file to test the method NCBIXML.handle()
+                            <?xml version="1.0"?>
+                            <!DOCTYPE BlastOutput PUBLIC "-//NCBI//NCBI BlastOutput/EN" "http://www.ncbi.nlm.nih.gov/dtd/NCBI_BlastOutput.dtd">
+                            <BlastOutput>
+                              <BlastOutput_program>tblastn</BlastOutput_program>
+                              <BlastOutput_version>TBLASTN 2.2.26+</BlastOutput_version>
+                              <BlastOutput_reference>Stephen F. Altschul, Thomas L. Madden, Alejandro A. Sch&amp;auml;ffer, Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997), &quot;Gapped BLAST and PSI-BLAST: a new generation of protein database search programs&quot;, Nucleic Acids Res. 25:3389-3402.</BlastOutput_reference>
+                              <BlastOutput_db>db/minirefseq_mrna</BlastOutput_db>
+                              <BlastOutput_query-ID>Query_1</BlastOutput_query-ID>
+                              <BlastOutput_query-def>random_s00</BlastOutput_query-def>
+                              <BlastOutput_query-len>32</BlastOutput_query-len>
+                              <BlastOutput_param>
+                                <Parameters>
+                                  <Parameters_matrix>BLOSUM62</Parameters_matrix>
+                                  <Parameters_expect>10</Parameters_expect>
+                                  <Parameters_gap-open>11</Parameters_gap-open>
+                                  <Parameters_gap-extend>1</Parameters_gap-extend>
+                                  <Parameters_filter>L;</Parameters_filter>
+                                </Parameters>
+                              </BlastOutput_param>
+                              <BlastOutput_iterations>
+                              </BlastOutput_iterations>
+                            </BlastOutput>
+                            """)
+        records = NCBIXML.parse(handle)
+        self.assertRaises(ValueError, next, records)
+        handle.close()
+
+    def test_xml_2226_tblastn_008_err(self):
+        """Parse TBLASTN 2.2.26+."""
+        handle = StringIO("")
+        records = NCBIXML.parse(handle)
+        self.assertRaises(ValueError, next, records)
+        handle.close()
+
+    def test_xml_2226_tblastn_009_err(self):
+        """Parse TBLASTN 2.2.26+."""
```

Comment:
This one is tricky.

As fuzzy test I've appended different XML bodies into one source file to test the fact that even multiple output gathered into one single source could be successfully parsed. 

Do you think that I should keep this edited XML and add it to the folder `Tests\PDB`?